Makefile

```diff
@@ -326,8 +326,6 @@ operator-lint: gowork ## Runs operator-lint
 # $oc delete -n openstack mutatingwebhookconfiguration/movs.kb.io
 SKIP_CERT ?=false
 .PHONY: run-with-webhook
-run-with-webhook: export OVS_IMAGE_URL_DEFAULT=quay.io/podified-antelope-centos9/openstack-ovn-base:current-podified
-run-with-webhook: export OVN_IMAGE_URL_DEFAULT=quay.io/podified-antelope-centos9/openstack-ovn-controller:current-podified
 run-with-webhook: manifests generate fmt vet ## Run a controller from your host.
```

Comment:
I think would be good to adapt make test too to this.

Reply:
Ah, sorry.  I didn't realize that we had integrate webhooks with the `envtest` stuff here.  I have adjusted the tests now.